Update the README to give a basic description of the Council and the purpose of the repo (#1)
rust-lang/leadership-council: README.md

# The Rust Project Leadership Council

This repo is the home for documentation related to the Leadership Council for the Rust Programming Language Project (a.k.a. the Council).

The Council was established by [RFC #3392][rfc]. The RFC describes the Council at a high-level as the following:

> > This RFC establishes a Leadership Council as the successor of the core team and the new governance structure through which Rust Project members collectively confer the authority to ensure successful operation of the Project. The Leadership Council delegates much of this authority to teams (which includes subteams, working groups, etc.3) who autonomously make decisions concerning their purviews. However, the Council retains some decision-making authority, outlined and delimited by this RFC.

Reviewer: ```suggestion
> This RFC establishes a Leadership Council as the successor of the core team and the new governance structure through which Rust Project members collectively confer the authority to ensure successful operation of the Project. The Leadership Council delegates much of this authority to teams (which includes subteams, working groups, etc.) who autonomously make decisions concerning their purviews. However, the Council retains some decision-making authority, outlined and delimited by this RFC.
```
Typo

>

> > The Council will be composed of representatives delegated to the Council from each top-level team.

Reviewer 2: ```suggestion
> The Council is composed of representatives delegated to the Council from each top-level team.
```

Author: This is a bit complicated is its meant to be a direct quote of the RFC, but I will change it to use brackets to indicate a change from the direct quote: "[is]"
